pom.xml

```diff
@@ -2880,6 +2880,7 @@
                                     -Xep:OptionalNotPresent:ERROR \
                                     -Xep:OrphanedFormatString:ERROR \
                                     -Xep:Overrides:ERROR \
+                                    -Xep:PatternMatchingInstanceof:ERROR \
```

Comment:
This doesn't trigger on anything? When I rebased I got some hits, for example here:

https://github.com/trinodb/trino/blob/96a476d127404af07a44876d91812043a5b02115/client/trino-client/src/main/java/io/trino/client/ResultRowsDecoder.java#L86

Reply:
The client is JDK 8 so it can't be used :)